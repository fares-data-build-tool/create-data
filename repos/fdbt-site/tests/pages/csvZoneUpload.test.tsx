import * as React from 'react';
import { shallow } from 'enzyme';
import CsvZoneUpload from '../../src/pages/csvZoneUpload';

describe('pages', () => {
    describe('csvzoneupload', () => {
        it('should render correctly', () => {
            const tree = shallow(
                <CsvZoneUpload
                    csvUploadApiRoute=""
                    csvUploadTitle="Upload fare zone as CSV"
                    csvUploadHintText=""
                    guidanceDocDisplayName=""
                    guidanceDocAttachmentUrl=""
                    guidanceDocSize=""
                    csvTemplateDisplayName=""
                    csvTemplateAttachmentUrl=""
                    csvTemplateSize=""
                    errors={[]}
                    csrfToken=""
                />,
            );
            expect(tree).toMatchSnapshot();
        });
    });
});
