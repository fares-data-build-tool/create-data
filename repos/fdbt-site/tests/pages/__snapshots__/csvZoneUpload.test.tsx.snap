// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages csvzoneupload should render correctly 1`] = `
<Layout
  description="CSV Zone Upload page of the Fares data build tool"
  title="CSV Zone Upload Method - Fares data build tool"
>
  <main
    className="govuk-main-wrapper app-main-class"
    id="main-content"
    role="main"
  >
    <UserDataUploadComponent />
  </main>
</Layout>
`;
